### DISCUSSION TITLE
How to use the software when there are hands in the video?

### DISCUSSION THREAD

**Discussion by div0rce** [svsdval/video2midi#57] [Q&A]
My question is as the title says. I enjoy using the software for Synthesia videos, however I am unsure how to use the software in videos where you can see the pianists hands along with the sparks on screen. Let alone, is it even possible? I checked a past discussion and I saw pianist hands, so I figured it is; I personally just don't know how to do it.
![image](https://github.com/user-attachments/assets/3a1e3d85-f085-4672-86bd-8b0cc1b3e4d6)
To be frank, I don't really know how to use the software all too well. All I really know how to do is align the piano and click start on videos where it's just Synthesia. If I could get some help on which settings I need to utilize, that would be all very appreciated. Additionally, I request some patience because I don't really know how the settings work -- I wish there was documentation on what everything does, how changing it effects the output, and how to properly tweak the settings.

**Comment by svsdval** [comment]
Documentation: https://github.com/svsdval/video2midi/wiki/Manual-for-functions
If you didn't find something in the documentation, write to me, I will complete it with examples.
Here is a video for your case: https://www.youtube.com/watch?v=DGjKDQc8LMM
